synfig-studio/src/gui/dialogs/canvasresize.cpp

```diff
@@ -195,7 +210,11 @@ void CanvasResize::on_action_signal_response(int response_id)
 		refresh_content();
 		break;
 	case OKAY:
-		if (new_width == old_width && new_height == old_height) break;
+		if (new_width == old_width && new_height == old_height)
+			if (is_toggled == was_toggled && is_toggle_unique) break;
```

Comment:
`is_toggled` is of type "bool" and `was_toggled` is an "int". Is this intended ? 

c and cpp are quite permissive about comparisons of bools and ints but in general I think it should be better avoided probably unless absolutely needed.

Reply:
To be on the safe side we could cast `was_toggled` to `bool`

Reply:
How about just making `was_toggled` itself a boolean if it's only going to be used as such ?